Mage/src/main/java/mage/abilities/DelayedTriggeredAbilities.java

```diff
@@ -27,11 +27,9 @@ public void checkTriggers(GameEvent event, Game game) {
         // TODO: add same integrity checks as TriggeredAbilities?!
         for (Iterator<DelayedTriggeredAbility> it = this.iterator(); it.hasNext(); ) {
             DelayedTriggeredAbility ability = it.next();
-            if (ability.getDuration() == Duration.Custom) {
```

Comment:
It's very important changes (force to discard all delayed triggers, not only custom duration). And can potentially broke bad writing abilities (delayed abilities that uses stack instead non-stack). See more details in https://github.com/magefree/mage/issues/12111#issuecomment-2118922969

It must be researched as part of #12111 (is it a real problem or not).

Reply:
Also must add docs on usage of stack and non-stack delayed triggers with custom and non-custom duration from #12111:

![shot_240608_231746](https://github.com/magefree/mage/assets/8344157/5db1d946-876f-4d7c-a253-2f3c599c1ba4)